lib/client/boluscalc.js

```diff
@@ -621,41 +621,41 @@ function init(client, $) {
   boluscalc.loadFoodDatabase = function loadFoodDatabase(event, callback) {
     categories = [];
     foodlist = [];
-    var records = client.sbx.data.food || [];
-    records.forEach(function (r) {
-      if (r.type == 'food') {
-        foodlist.push(r);
-        if (r.category && !categories[r.category]) {
-          categories[r.category] = {};
-        }
-        if (r.category && r.subcategory) {
-          categories[r.category][r.subcategory] = true;
-        }
+    $.ajax('/api/v1/food/regular.json', {
```

Comment:
This solves another issue. The fact that BolusWizard -->  Quickpick was not fetching food entries.

Reply:
There are hardcoded english since Google Assistant has limited voice language support. One using an unsupported language would result in errors.

Text support can be added within DialogFlow by each user, by editing/adding the intents. 

In the template I created I intentionally left some custom payload for Telegram in English to be changed in any language.

Reply:
Did not know of the speech plugin. It would be useful to add this to the DialogFlow payload.

Reply:
@PieterGit 

Yes the plugin is not called when not ENABLED.